WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java

```diff
@@ -658,9 +658,9 @@ public static boolean isGutenbergDefaultForNewPosts() {
         return getBoolean(DeletablePrefKey.GUTENBERG_DEFAULT_FOR_NEW_POSTS, false);
     }
 
-    public static void setGutenbergDefaultForNewPosts(boolean defaultForNewPosts) {
-        AnalyticsTracker.track(defaultForNewPosts ? Stat.EDITOR_GUTENBERG_ENABLED : Stat.EDITOR_GUTENBERG_DISABLED);
```

Comment:
With this Track firing removal, it seems that we never trigger the `EDITOR_GUTENBERG_ENABLED` track. Perhaps we need to move it to the site setting switching?

Reply:
cc @etoledom so to be in alignment with the WPiOS side.

Reply:
Oh right! 🤦‍♂ Re-added in ee57eb45afeff9daa98df47291e752ab6d3c65ff

Also tracking with site details now.

Reply:
Looping back, added a related comment here https://github.com/wordpress-mobile/WordPress-Android/pull/10254/commits/ee57eb45afeff9daa98df47291e752ab6d3c65ff#r307692184.